frameworks/elastic/universe/config.json

```diff
@@ -696,6 +696,41 @@
           "type": "boolean",
           "default": true
         },
+        "searchguard_enabled": {
+          "description": "Set it to true to enable the Search Guard plugin for Elastic. Defaults to false.",
+          "type": "boolean",
+          "default": false
+        },
+        "searchguard_internal_users": {
+          "type": "string",
+          "description": "The path in the DC/OS Secret Store to retrieve the Searchguard internal user database.",
+          "default": ""
+        },
+        "searchguard_config": {
+          "type": "string",
+          "description": "The path in the DC/OS Secret Store to retrieve the Searchguard authenticators and authorisation backends configuration.",
+          "default": ""
+        },
+        "searchguard_action_groups": {
+          "type": "string",
+          "description": "The path in the DC/OS Secret Store to retrieve the Searchguard named permission groups.",
+          "default": ""
+        },
+        "searchguard_roles": {
+          "type": "string",
+          "description": "The path in the DC/OS Secret Store to retrieve the Searchguard defined roles and the associated permissions.",
+          "default": ""
+        },
+        "searchguard_roles_mapping": {
+          "type": "string",
+          "description": "The path in the DC/OS Secret Store to retrieve the Searchguard mapping of backend roles, hosts and users to roles.",
+          "default": ""
+        },
+        "searchguard_tenants": {
+          "type": "string",
+          "description": "The path in the DC/OS Secret Store to retrieve the Searchguard tenants for configuring Kibana access.",
+          "default": ""
+        },
```

Comment:
@alexeygorobets what do you think about having all "Search Guard"-related settings under `elasticsearch.searchguard.*`? E.g.:
```json
{
  "elasticsearch": {
    "searchguard": {
      "enabled": true,
      ...
    }
  }
}